eel_hole/templates/partials/navbar.html

```diff
@@ -14,14 +14,15 @@ <h1 class="title my-auto px-2 logo">pudl</h1>
                 conventions</a>
             <a class="navbar-item" href="https://youtu.be/-mU7QwAZY8c">Tutorial video</a>
             <a class="navbar-item" href="https://forms.gle/ohzyKRoQutqxY3WZ6">Submit feedback</a>
+            <a class="navbar-item" href="/privacy-policy">Privacy policy</a>
         </div>
         <div class="navbar-end">
             {% if current_user.is_authenticated %}
             <a class="button mx-2 my-1 py-2" href="/logout">
                 Log out of {{ current_user.username }}
             </a>
             {% else %}
-            <a class="button mx-2 my-1 py-2 is-primary" href="/login">
+            <a class="button mx-2 my-1 py-2 is-primary" href="/login?next={{request.full_path}} ">
```

Comment:
I'm finding that the `q` param is either getting stripped or just not passed in. So this drops you off back at `/search` but doesn't retain your query.

Might be something to do with how I'm using HTMX to provide active search - which changes the URL on the client side, but doesn't re-render this login button in the template. We'd have to set up some sort of special button handler here, which to me is firmly in "rabbit hole that shouldn't block this PR" territory.